src/main/java/hockeyapp/HockeyappApplication.java

```diff
@@ -46,7 +48,8 @@
     @DataBoundConstructor
     public HockeyappApplication(String apiToken, String appId, boolean notifyTeam,
                                 String filePath, String dsymPath, String libsPath,
-                                String tags, boolean downloadAllowed, OldVersionHolder oldVersionHolder,
+                                String tags, String teams, boolean mandatory, 
+                                boolean downloadAllowed, OldVersionHolder oldVersionHolder,
```

Comment:
I'm thinking more about how “filePath, dsymPath, libsPath” and “appToken, appId” was grouped together in the method params. I can revert the changes if you want?